### ISSUE TITLE
Vehicle VIN number contains illegal characters

### ISSUE THREAD

**Issue by DanielLipowicz** [Marak/faker.js#1062]
Based on online sources VIN number should not contain [O,I,Q] characters. Because of risk of mistake O with 0, I with 1 and q with 9.
 https://en.wikipedia.org/wiki/Vehicle_identification_number
Currently there are a test for a VIN
```
  describe("vin()", function () {
      it("returns valid vin number", function () {
        var vin = faker.vehicle.vin();
        assert.ok(vin.match(/^[A-Z0-9]{10}[A-Z]{1}[A-Z0-9]{1}\d{5}$/));
      });
  });
```
Above code allow O, I, Q characters in VIN number

**Comment by DanielLipowicz** [comment]
I created new MR to fix this issue - https://github.com/Marak/faker.js/pull/1063/files

**Comment by Marak** [comment]
Good catch @DanielLipowicz 

Thank you for your contributions!

This should be resolved with merging #1063
